src/seedu/addressbook/AddressBook.java

```diff
@@ -480,14 +481,31 @@ private static String getMessageForPersonsDisplayedSummary(
 	private static ArrayList<HashMap<PersonProperty, String>> getPersonsWithNameContainingAnyKeyword(
```

Comment:
You could explicitly mention about the case-insensitivity in the header comments for this function.